.github/workflows/generate_pipeline.yaml

```diff
@@ -0,0 +1,45 @@
+name: Generate Pipeline
+
+on:
+  push:
+    branches:
+      -  development
+      -  test
+
+jobs:
+  generate-pipeline:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests jinja2 pathlib aixplain
+
+      - name: Run generate.py to create pipeline.py
+        working-directory: aixplain/modules/pipeline
+        env:
+          BACKEND_URL: https://test-platform-api.aixplain.com
```

Comment:
These URLs may be updated. If it is too difficult to update with the one corresponding with the environtment, consider the production URL as the one to add here.